css-view-transitions-1/Overview.bs

```diff
@@ -1204,7 +1210,9 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 		1. [=Capture the old state=] for |transition|.
 
-			If failure is returned, then [=skip the view transition=] for |transition| with an "{{InvalidStateError}}" {{DOMException}} in |transition|'s [=relevant Realm=],
+		1. If |transition|'s [=ViewTransition/process old state captured=] is not null, then call [=ViewTransition/process old state captured=] and return.
```

Comment:
We need to inform the "process old state captured algorithm" if there is a failure in "capture the old state".  You could set the phase to done for it.

Reply:
Done